README.md

```diff
@@ -41,6 +42,12 @@ Facebook has adopted a Code of Conduct that we expect project participants to ad
 
 Read our [contributing guide](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md) to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes to Docusaurus.
 
+### Contribute online with a single click
+
+You can use Gitpod(A free online VS Code like IDE) for contributing with a single click it will launch a ready to code workspace with all the dependencies being pre-installed and the build & start script in process so that you can start contributing straight away.
```

Comment:
```suggestion
You can use Gitpod (a free, online, VSCode-based IDE) for contributing.  It will launch a ready to code workspace with all the dependencies installed and the build script in progress so that you can start contributing straight away.
```